### DISCUSSION TITLE
Create metadata without downloading video

### DISCUSSION THREAD

**Discussion by ilkersigirci** [jmbannon/ytdl-sub#265] [Q&A] [answered]
Hi, 
Firstly thanks for this awesome tool. Finally I am successfully watch my youtube videos on my jellyfin server with appropriate metadata. However, I have already downloaded GBs of youtube videos which don't have any .nfo tags hence they can't be correctly shown in the jellyfin. I have read docs and the repo wiki but I haven't found any option for downloading only metadata of the given youtube video link. In the [discussion](https://github.com/jmbannon/ytdl-sub/issues/166#issuecomment-1212317922),  @jmbannon said that we can download those videos with low quality and after creating metadata we can delete those video files. But the problem with this is that some of my downloaded youtube videos are too long(9+ hours) and even downloading them with lower quality takes too much time. I would be happy to know whether there are  any solution for this? Thanks again

**Comment by jmbannon** [comment, marked as answer]
No known solution right now other than what I suggested. You could try only download a portion of the video using `ytdl_options`: https://www.reddit.com/r/youtubedl/comments/q7woof/whats_the_new_command_for_downloading_parts_of_a/

You might be better off redownloading the missing nfo channels using these new built-in presets:
https://ytdl-sub.readthedocs.io/en/latest/presets.html#tv-shows

They're somewhat experimental, but FWIW I'm using them with no issues so far. They are meant to be the 'standard' way to download things in TV show format.

**Reply by ilkersigirci** [reply]
Thanks, I will try the second approach that you suggested
